src/js/stores/NewsFeedStore.jsx

```diff
@@ -79,19 +115,79 @@ class NewsFeedStore extends EventEmitter {
       contentType:"application/json",
       cache: false,
       success: function(response) {
-        dispatcher.dispatch({type: "NEWSFEED_POST", post: response["id"]});
+        console.log("THIS ACTUALLY FINISHED");
```

Comment:
(That applies to almost all of those console statements. They'll be cleaned up as soon as the photo posting is functional)